### PR TITLE
feat: render examples in module docs

### PR DESCRIPTION
**Pull request by crowlKats**

### CODE REVIEW

File: doc/jsdoc.tsx [hunk 1]
```diff
@@ -17,13 +17,13 @@ export function JsDoc(
...
 }
 
-export function JsDocModule(
-  { children, context }: {
-    children: Child<DocNodeTupleArray<DocNodeModuleDoc>>;
-    context: Context;
-  },
-) {
-  const moduleDoc = take(children, true, true);
+export function getJsDocModule(
```
Comment (kt3k) [inline doc/jsdoc.tsx:20]: nit: How about moving this to util.ts?

**Review by kt3k — APPROVED** [review]
LGTM

File: doc/jsdoc.tsx (hunk 1 above)
Comment (crowlKats) [reply doc/jsdoc.tsx:20]: yea was thinking about that; will do it. will also move the JsDoc function to the markdown file; shouldnt need to have its own file for like 10 lines of code

Comment (crowlKats) [reply doc/jsdoc.tsx]: @kt3k I decided to remove that function completely as its used in a single place. while i was at it i moved the jsdoc component to the markdown file.

**Pull request merged**